### PR TITLE
Update Goreleaser

### PR DESCRIPTION
**Pull request by SuperQ**
* Update to latest goreleaser version.
* Add checksums.txt file to the release.
* Add /etc/default environment file to make flag changes easier.
* Fix up .gitignore to be more specific about ignoring the built binary.

Signed-off-by: Ben Kochie <superq@gmail.com>

### CODE REVIEW

**Comment by SuperQ** [comment]
Huh, sill no circleci.

**Comment by ncabatoff** [comment]
Strange.  I thought it was covered by these settings, any ideas what else might be needed?

image

**Comment by ncabatoff** [comment]
Hmm, well I didn't have the "Pull Request" option enabled in the GH/circleci webhook settings, so maybe that was it.  Would you mind trying to open a new PR to see if that did the trick?

**Comment by SuperQ** [comment]
Pushed a simple yaml change, now it's working.

**Comment by SuperQ** [comment]
I removed the docker publish on every CI run. If there's a desire to have an image generated for every merge to master, I could add that back as a separate workflow entry.

**Comment by ncabatoff** [comment]
> I removed the docker publish on every CI run. If there's a desire to have an image generated for every merge to master, I could add that back as a separate workflow entry.

That would be nice, not essential though, and probably a pain for you to do since you don't have the creds to see it working.  One thing I worry about with the current state of the PR is that it doesn't look like the `latest` image tag would ever be populated now?

**Comment by SuperQ** [comment]
Oh yea, I can put the `:latest` tag onto the release workflow.

File: .circleci/config.yml [hunk 1]
```diff
@@ -25,7 +34,7 @@ jobs:
       - run: docker build -t $IMAGE_NAME:latest .
       - run: |
           echo "$DOCKERHUB_PASSWORD"| docker login -u $DOCKERHUB_USERNAME --password-stdin
-          docker push $IMAGE_NAME:latest
+          docker push $IMAGE_NAME:master
```
Comment (ncabatoff) [inline .circleci/config.yml:37]: I don't see where we create an image with the tag `master`, so I think this would fail.

Comment (SuperQ) [reply .circleci/config.yml:37]: Oh, right, I need to update the build in that job.

**Comment by ncabatoff** [comment]
Thanks again!

**Pull request merged**